### PR TITLE
Issues/153 docker image with data insert script

### PR DESCRIPTION
**Pull request by janherza**

### CODE REVIEW

File: powerauth-test-server/DockerfileExtended [hunk 1]
```diff
@@ -0,0 +1,54 @@
+FROM tomcat:jdk11-adoptopenjdk-openj9
```
Comment (romanstrobl) [inline powerauth-test-server/DockerfileExtended:1]: I am not sure we should use `powerauth-test-server/DockerfileExtended`. Dockerfiles are always called `Dockerfile`.

File: powerauth-test-server/DockerfileExtended [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+LABEL maintainer="roman.strobl@wultra.com"
+
+# Prepare environment variables
+ENV JAVA_HOME /opt/java/openjdk
+ENV TOMCAT_HOME /usr/local/tomcat
+
```
Comment (romanstrobl) [inline powerauth-test-server/DockerfileExtended:7]: Extra newline.

**Review by romanstrobl — APPROVED** [review]
I have one concern related to the Dockerfile name, otherwise I think this is OK for testing.

File: powerauth-test-server/docker/powerauth-test-server.xml [hunk 3]
```diff
@@ -29,4 +29,11 @@
     <!-- Additional JPA Configuration -->
     <Parameter name="spring.jpa.database-platform" value="${POWERAUTH_TEST_SERVER_JPA_DATABASE_PLATFORM:org.hibernate.dialect.PostgreSQL95Dialect}"/>
 
+    <!-- Liquibase Script Configuration -->
+    <Parameter name="spring.liquibase.enabled" value="${LIQUIBASE_ENABLED}:true"/>
```
Comment (banterCZ) [inline powerauth-test-server/docker/powerauth-test-server.xml:33]:
```suggestion
    <Parameter name="spring.liquibase.enabled" value="${LIQUIBASE_ENABLED:true}"/>
```

File: powerauth-test-server/src/main/resources/application.properties [hunk 4]
```diff
@@ -31,3 +31,10 @@ spring.jpa.hibernate.ddl-auto=update
 
 # Disable open session in view to avoid startup warning of Spring boot
 spring.jpa.open-in-view=false
+
+# Liquibase Changelog Configuration
+spring.liquibase.enabled=true
```
Comment (banterCZ) [inline powerauth-test-server/src/main/resources/application.properties:36]:
```suggestion
spring.liquibase.enabled=${LIQUIBASE_ENABLED:true}
```

File: powerauth-test-server/docker/powerauth-test-server.xml [hunk 5]
```diff
@@ -29,4 +29,11 @@
...
+    <Parameter name="spring.liquibase.parameters.PA_CLOUD_SERVICE_BASE_URL" value="${POWERAUTH_CLOUD_SERVICE_BASE_URL}"/>
```
Comment (banterCZ) [inline powerauth-test-server/docker/powerauth-test-server.xml:34]: Let's keep consistent case.
```suggestion
    <Parameter name="spring.liquibase.parameters.pa-cloud-service-base-url" value="${POWERAUTH_CLOUD_SERVICE_BASE_URL}"/>
```

File: powerauth-test-server/liquibase/data/config-data/postgresql/insert_data.sql [hunk 6]
```diff
@@ -0,0 +1,64 @@
+-- pa cloud property
+INSERT INTO pa_cloud_property (name, value)
+    VALUES ('service.base.url', '${PA_CLOUD_SERVICE_BASE_URL}')
```
Comment (banterCZ) [inline powerauth-test-server/liquibase/data/config-data/postgresql/insert_data.sql:3]:
```suggestion
    VALUES ('service.base.url', '${spring.liquibase.parameters.pa-cloud-service-base-url}')
```

File: powerauth-test-server/src/main/resources/application.properties [hunk 7]
```diff
@@ -31,3 +31,10 @@ spring.jpa.hibernate.ddl-auto=update
...
+spring.liquibase.change-log=classpath:liquibase/data/changelog.xml
+spring.liquibase.parameters.PA_CLOUD_SERVICE_BASE_URL=
```
Comment (banterCZ) [inline powerauth-test-server/src/main/resources/application.properties:38]:
```suggestion
spring.liquibase.parameters.pa-cloud-service-base-url=${PA_CLOUD_SERVICE_BASE_URL:}
```

File: powerauth-test-server/liquibase/data/config-data/postgresql/insert_data.sql [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+INSERT INTO pa_operation_template (id, template_name, operation_type, data_template, signature_type,
+    max_failure_count, expiration)
+    SELECT nextval('pa_operation_template_seq'), 'login', 'login', 'A2', 'possession_knowledge, possession_biometry',
+        5, 300
+    WHERE NOT EXISTS (SELECT * FROM pa_operation_template WHERE template_name = 'login' AND operation_type = 'login');
+
+INSERT INTO pa_operation_template (id, template_name, operation_type, data_template, signature_type,
+    max_failure_count, expiration)
+    SELECT nextval('pa_operation_template_seq'), 'payment', 'authorize_payment',
+        'A1*A${amount}${currency}*I${iban}', 'possession_knowledge, possession_biometry', 5, 300
```
Comment (banterCZ) [inline powerauth-test-server/liquibase/data/config-data/postgresql/insert_data.sql:42]: What about `A1*A${amount}${currency}*I${iban}`, is it kept?

File: powerauth-test-server/DockerfileExtended (hunk 1 above)
Comment (janherza) [reply powerauth-test-server/DockerfileExtended]: Finally, this file will be deleted.

File: powerauth-test-server/DockerfileExtended (hunk 2 above)
Comment (janherza) [reply powerauth-test-server/DockerfileExtended]: Finally, this file will be deleted.

**Pull request merged**